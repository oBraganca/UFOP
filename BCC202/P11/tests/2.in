1 jose 5 1 5 1990
1 guilherme 6 9 3 1998
1 valentina 9 1 3 1945
1 jesus 10 25 1 0000
1 pedro 1 19 10 1930
1 maria 2 17 5 1972
1 giovana 7 8 3 2015
1 siva 5 3 11 1968
1 daiane 3 24 12 1993
1 paulo 4 6 4 1998
2
3
