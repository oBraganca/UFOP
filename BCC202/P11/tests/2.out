10 | 25/01/0000 | jesus
09 | 01/03/1945 | valentina
07 | 08/03/2015 | giovana
06 | 09/03/1998 | guilherme
05 | 03/11/1968 | siva
05 | 01/05/1990 | jose
04 | 06/04/1998 | paulo
03 | 24/12/1993 | daiane
02 | 17/05/1972 | maria
01 | 19/10/1930 | pedro
